components/forms/Login/index.jsx

```diff
@@ -0,0 +1,42 @@
+import Link from 'next/link';
+
+const Login = () => {
+  return (
+    <form className="w-80">
+      <div className="h-24 mb-5">
```

Comment:
```suggestion
      <div className="h-24 mb-6">
```

This would change the bottom margin from `20px` to `24px` as per the Figma design ;-)

Same suggestion for the Password component input group below ;-)

Reply:
![image](https://user-images.githubusercontent.com/77609814/160320431-d84102c6-4f3a-48b6-96cd-5b072e0b809a.png)

Reply:
sure, done in https://github.com/tyrro/react-survey/pull/50/commits/42a749da2be9185adad4215698e02f740cb23b2f